test/download-test.js

```diff
@@ -12,52 +12,17 @@ var build = path.join(__dirname, 'build')
 var unpacked = path.join(build, 'Release/leveldown.node')
 
 test('downloading from GitHub, not cached', function (t) {
-  t.plan(14)
+  t.plan(10)
   rm.sync(build)
   rm.sync(util.prebuildCache())
 
   var opts = getOpts()
   var downloadUrl = util.getDownloadUrl(opts)
   var cachedPrebuild = util.cachedPrebuild(downloadUrl)
-  var npmCache = util.npmCache()
   var tempFile
 
-  var existsCallNum = 0
-  var _access = fs.access ? fs.access.bind(fs) : fs.access
-  var _exists = fs.exists.bind(fs)
-  if (_access) {
-    fs.access = function (path, a, cb) {
-      if (existsCallNum++ === 0) {
-        t.equal(path, npmCache, 'fs.exists called for npm cache')
-        _access(path, cb)
-      }
-    }
-  }
-  fs.exists = function (path, cb) {
-    if (existsCallNum++ === 0) {
-      t.equal(path, npmCache, 'fs.exists called for npm cache')
-      _exists(path, cb)
-    } else {
-      t.equal(path, cachedPrebuild, 'fs.exists called for prebuild')
-      _exists(path, function (exists) {
-        t.equal(exists, false, 'prebuild should be cached')
-        cb(exists)
-      })
-    }
-  }
-
-  var mkdirCount = 0
-  var _mkdir = fs.mkdir.bind(fs)
-  fs.mkdir = function () {
-    var args = [].slice.call(arguments)
-    if (mkdirCount++ === 0) {
-      t.equal(args[0], util.prebuildCache(), 'fs.mkdir called for prebuildCache')
-    }
-    _mkdir.apply(fs, arguments)
-  }
```

Comment:
I removed these tests. They merely checks that these calls were made, which basically tests internal functionality. We could remove more of these for `.create{Read,Write}Stream` as well, although some of them are useful.

Let me know what you guys think.